### PR TITLE
Tailor feature

### PR DESCRIPTION
**Pull request by wassimchakib**
**In this feature, I added:** 

• Home page
• About page
• Followed design guidelines by using: the same colors, same font, and spacing
• Designed the website to be mobile first and media query to adapt it to the desktop version
• Added link interactions to move from home to about page and vice versa
• Added Section "Testimonials" dynamically by using JS object
• Added `more button` to the mobile version so It can be shrunk.
For more information, you can see my video here 👍 [video](https://www.loom.com/share/ae10b390f4714145ad42528e2c919ccc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# Capstone-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone-Project
+
+> This project is about creating a website for a company that provides the best tailoring services.
+
```
Comment (andgarzonmal) [inline README.md:6]: - [x]  Kyndly add the required loom video 📽️📺 on the pull request description accorging to the project requirements.

image

image

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Learning to code is learning to create and innovate
```
Comment (andgarzonmal) [inline README.md:64]: - [x]  Kyndly add to the "Acknowledgements" section the appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👌👌

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @wassimchakib    ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @wassimchakib, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on finishing this module! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] I suggest that you also include the link to the design itself as well in the acknowledgment section just so it's easy to find the design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**